server/base/src/main/java/org/apache/accumulo/server/compaction/FileCompactor.java

```diff
@@ -280,7 +280,10 @@ public CompactionStats call() throws IOException, CompactionCanceledException {
           }
         }
       } catch (IOException | RuntimeException e) {
-        log.warn("{}", e.getMessage(), e);
+        log.warn("{}", e.getMessage());
+        if (log.isDebugEnabled()) {
+          log.debug(e.getMessage(), e);
```

Comment:
This is a safer version, in case there's any braces in the message that could be misinterpreted as a string interpolation.

```suggestion
          log.debug("{}", e.getMessage(), e);
```

Reply:
Good to know, I was wondering what the purpose of that was. I will keep that in mind for the future.